tendermint/tests/serialization.rs

```diff
@@ -57,8 +57,8 @@ fn deserialize_integer_from_string() {
 
     let result: IntegerTests = serde_json::from_str(&incoming).unwrap();
 
-    assert_eq!(result.unsigned, 9007199254740992);
-    assert_eq!(result.signed, -9007199254740994);
+    assert_eq!(result.unsigned, 9_007_199_254_740_992);
+    assert_eq!(result.signed, -9_007_199_254_740_994);
```

Comment:
For some reason clippy doesn't run as strictly as it used to be on CI. There a few places where it complained when run with:
`cargo clippy --all --all-targets -- -Dwarnings -Drust-2018-idioms`